### PR TITLE
Integrate login page with login api and add check on login api

### LINKED ISSUE #61: Integrate login API in login page
- After login API successful response store token in cookies and redirect to homepage

### PR DESCRIPTION
**Pull request by dewa-himanshu24**
closes #61

### CODE REVIEW

File: pages/Login.tsx [hunk 1]
```diff
@@ -58,4 +128,8 @@ const Login = () => {
   )
 }
 
-export default Login
\ No newline at end of file
+export default Login
+
+function setTokenCookie(token: any) {
+  throw new Error('Function not implemented.');
+}
```
Comment (jainendra) [inline pages/Login.tsx]: This is not required

File: pages/Login.tsx [hunk 2]
```diff
@@ -1,10 +1,76 @@
...
+    }
+  }
+
+  const [enteredEmail, setEnteredEmail] = useState("")
+  const [enteredPassword, setEnteredPassword] = useState("")
+
+  // if cookie redirect to homepage.....
+  useEffect(() => {
+
+    if (getCookie('xBhaktToken')) {
```
Comment (jainendra) [inline pages/Login.tsx]: You can use `hasCookie('<cookie-name>')` method here. More details about the function here - https://github.com/andreizanik/cookies-next

File: pages/Login.tsx (hunk 1 above)
Comment (dewa-himanshu24) [reply pages/Login.tsx]: forgot to remove these lines of code,
Done

File: pages/Login.tsx (hunk 2 above)
Comment (dewa-himanshu24) [reply pages/Login.tsx]: Done

File: pages/api/hello.ts [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
+      res.status(200).json({ name: 'John Doe' })
+    } else {
+      throw Error('No data on decode');
+    }
+  } catch (err) {
+    const message = 'Error while validating token';
+    console.log(`${message}: ${err}`);
+    res.status(400).json({ message: message });
+  }
 }
```
Comment (jainendra) [inline pages/api/hello.ts]: This is not related to this PR, please remove it

**Review by jainendra — APPROVED** [review]
